pkg/controller/admin/users_operations.go

```diff
@@ -78,7 +78,7 @@ func (c *Controller) HandleSystemAdminCreate() http.Handler {
 
 			// User does not exist, create a new one.
 			user = &database.User{
-				Name:  form.Email,
+				Name:  form.Name,
```

Comment:
This is a heinous bug. Good thing we are testing.

Reply:
yikes.

Reply:
Clicky clicky for-the-win!